src/components/v5/common/Fields/Datepicker/Datepicker.tsx

```diff
@@ -0,0 +1,173 @@
+import clsx from 'clsx';
```

Comment:
Is this file still needed after #1929 merge?

Reply:
It's not, you're right. I forgot to remove it after the merge.